tests/integration/test_draw_realisations_mesmer_newcodepath.py

```diff
@@ -294,44 +273,24 @@ def test_make_realisations(
     )
 
     global_variability = map_over_datasets(
-        xr.Dataset.rename, global_variability, {"samples": "tas"}
+        xr.Dataset.rename, global_variability, {"samples": "tas_resids"}
```

Comment:
```suggestion
        lambda ds: ds.rename({"samples": "tas_resids"}), global_variability
```

let's align this with the other uses